### ISSUE TITLE
Option to highlight guide for contained block

### ISSUE THREAD

**Issue by ma53** [SirTori/indenticator#14]
I would find it very useful to have the option to highlight the guide for the block contained by the current line (when present) instead of or in addition to (in a different style or color) the guide for the block containing the current line.

![image](https://user-images.githubusercontent.com/7172504/33432946-de3807f8-d5a7-11e7-8b41-62527352f448.png)

I did see #11, but didn't want to resurrect it since I'm not reporting a bug but requesting a feature.

**Comment by SirTori** [comment]
sorry for the late response.

Thats a nice idea and I'll look into it for the next release.

But since I am currently pretty busy I won't make any promises on when that release will happen.

**Comment by SirTori** [comment]
Feature has been released in 0.5.0
If you have any further questions or suggestions feel free to share them

**Comment by john-cj** [comment]
@SirTori Thank you very much for this option.

Personally, I use this:

``` json
"indenticator.inner.showHighlight": true, // Enable inner line
"indenticator.style": "none" // Disable outer line
```
